sdk/translation/azure-ai-translation-text/pom.xml

```diff
@@ -0,0 +1,114 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-ai-translation-text</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-ai-translation-text;current} -->
+
+  <name>Microsoft Azure client library for Text Translation</name>
+  <description>This package contains Microsoft Azure Text Translation client library.</description>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <tag>HEAD</tag>
+  </scm>
+
+  <licenses>
+    <license>
+      <name>The MIT License (MIT)</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <properties>
+    <jacoco.skip>true</jacoco.skip>
```

Comment:
Can I check what the coverage is right now? Do I need to remove this property and push it to ADO? Or what would be the best way to check it?

Reply:
Determining coverage can be done locally by removing `<jacoco.skip>true</jacoco.skip>` and running `mvn verify` on the project. When `jacoco.skip` is true no coverage is gathered, which is why it will need to be removed. `mvn verify` will run tests and have Jacoco gather coverage and generate a report. The report will be found in `/target/site/test-coverage`.